api/remote.go

```diff
@@ -23,7 +23,14 @@ type RemoteEndpoints interface {
 type RemoteEngine interface {
 	MaxT() int64
 	MinT() int64
+
+	// The external labels of the remote engine.
```

Comment:
Maybe we can elaborate a bit more what these two label sets can be used for?

Reply:
added a bit more commentary